### PR TITLE
rhbz1197902 - clear hibernate session after each translation batch save

### PR DESCRIPTION
**Pull request by huangp**
https://bugzilla.redhat.com/show_bug.cgi?id=1197902

- [ ] Need to backport to release branch

### CODE REVIEW

**Comment by huangp** [comment]
@zanata-jenkins retest this please

File: zanata-war/src/main/java/org/zanata/async/AsyncTaskManager.java [hunk 1]
```diff
@@ -101,7 +101,7 @@ public void execute() {
                     taskFuture.set(returnValue);
                 } catch (Throwable t) {
                     taskFuture.setException(t);
-                    log.warn(
+                    log.error(
                             "Exception when executing an asynchronous task.", t);
```
Comment (seanf) [inline zanata-war/src/main/java/org/zanata/async/AsyncTaskManager.java]: This change is okay, but it really should have had its own commit with its own commit message.

Comment (seanf) [inline zanata-war/src/main/java/org/zanata/async/AsyncTaskManager.java]: Perhaps we should log the stacktrace as WARN, and log the exception.toString as ERROR.

**Comment by djansen-redhat** [comment]
:white_check_mark: Tested.
The difference in time is quite significant, generally between ten and twenty times faster. :+1:

**Pull request merged**